.github/workflows/tests.yml

```diff
@@ -0,0 +1,49 @@
+name: Run tests
+
+on:
+  push:
+    branches: ["*"]
+  pull_request:
+    branches: [master]
+
+jobs:
+  handlebars:
+    runs-on: ubuntu-latest
+    name: Coding style handlebars
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run coding style tests (handlebars)
+        run: docker build . -f ./Docker/djlint.Dockerfile -t lemverse/djlint
+      - name: Cleanup
+        if: always()
+        run: docker run lemverse/djlint
+  eslint:
+    runs-on: ubuntu-latest
+    name: Coding style javascript
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run coding style tests (javascript)
+        run: docker build . -f ./Docker/eslint.Dockerfile -t lemverse/eslint
+      - name: Cleanup
+        if: always()
+        run: docker run lemverse/eslint
+  eclint:
+    runs-on: ubuntu-latest
+    name: Coding style global
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run coding style tests (all files)
+        run: docker build . -f ./Docker/editorconfig.Dockerfile -t lemverse/editorconfig
+      - name: Cleanup
+        if: always()
+        run: docker run lemverse/editorconfig
+  stylelint:
+    runs-on: ubuntu-latest
+    name: Coding style stylesheets
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run coding style tests (scss)
+        run: docker build . -f ./Docker/stylelint.Dockerfile -t lemverse/stylelint
+      - name: Cleanup
+        if: always()
+        run: docker run lemverse/stylelint
```

Comment:
wouldn't it be simpler to have just one job that runs pre-commit?

Reply:
As in only one Docker with all the dependencies which only run the pre-commit ?
That could be, indeed, an option, but I’m not sure if it's good practise.

Reply:
as in

```
jobs:
  pre-commit:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v3
    - uses: actions/setup-python@v3
    - uses: pre-commit/action@v3.0.0
```